test/metric_value_benchmark/emf_test.go

```diff
@@ -90,7 +100,47 @@ func (t *EMFTestRunner) validateEMFMetric(metricName string) status.TestResult {
 		return testResult
 	}
 
-	if !isAllValuesGreaterThanOrEqualToZero(metricName, values) {
+	if len(values) == 0 {
+		return testResult
+	}
+
+	for _, v := range values {
+		if v != 5 {
+			return testResult
+		}
+	}
+
+	testResult.Status = status.SUCCESSFUL
+	return testResult
+}
+
+func validateEMFLogs(group, stream string) status.TestResult {
+	testResult := status.TestResult{
+		Name:   "emf-logs",
+		Status: status.FAILED,
+	}
+
+	rs := jsonschema.Must(emfMetricValueBenchmarkSchema)
+
+	validateLogContents := func(s string) bool {
+		return strings.Contains(s, "\"EMFCounter\":5")
+	}
+
+	now := time.Now()
+	ok, err := awsservice.ValidateLogs(group, stream, nil, &now, func(logs []string) bool {
+		if len(logs) < 1 {
+			return false
```

Comment:
Would it be better to return error here for the validator? Since its not clear the reason its failed between when its does not have logs and when its does not match the emf?

Reply:
No, I am not changing this to return an error.